doc/releases/v0.4.txt

```diff
@@ -1,3 +1,27 @@
+v0.4.1
+------
+
+This is a minor release with some important fixes. Additionally, PIMS is no
+longer a required dependency, which makes installation easier.
+
+
+API Changes
+~~~~~~~~~~~
+
+- PIMS (Python IMage Sequence) is no longer a required dependency (:issue:`492`)
+
+
+Enhancements
+~~~~~~~~~~~~
+
+- Improved performance of adaptive linking (:issue:`489`)
+
+
+Bug fixes
+~~~~~~~~~
+
+- Fix enable_numba on Py3.6 and MS Windows systems (:issue:`486`)
```

Comment:
Add:

Fix bug that caused adaptive search to fail after many iterations (:issue:`489`)